Ports/bzip2/package.sh

```diff
@@ -3,11 +3,11 @@ port=bzip2
 version=1.0.8
 files="https://sourceware.org/pub/bzip2/bzip2-${version}.tar.gz bzip2-${version}.tar.gz ab5a03176ee106d3f0fa90e381da478ddae405918153cca248e682cd0c4a2269"
 auth_type=sha256
-makeopts=("bzip2")
+makeopts+=("bzip2")
 installopts=("PREFIX=${SERENITY_INSTALL_ROOT}/usr/local")
 
 build() {
-    run make CC="${CC}" "${makeopts[@]}" bzip2
+    run make CC="${CC}" AR="${AR}" RANLIB="${RANLIB}" "${makeopts[@]}"
```

Comment:
Where is `AR` and `RANLIB` coming from? Those were not there before as far as I can tell.

Reply:
Whoops, I think this is an artifact of me doing multiple unrelated changes simultaneously. Good catch, will get rid of.